README.md

```diff
@@ -1,3 +1,15 @@
 # microcosm-flask
 
-Opinionated Flask services
+Opinionated Flask services.
+
+[![Circle CI](https://circleci.com/gh/globality-corp/microcosm-flask/tree/develop.svg?style=svg)](https://circleci.com/gh/globality-corp/microcosm-logging/tree/develop)
+
+
+## Conventions
+
+ - Every Flask service has a health check endpoint at `/api/health`
+
+
+## Configuration
+
+ - The object graph's `debug` and `testing` flags are propagated to the Flask application.
```

Comment:
You can set `flask.debug` and `flask.testing` when you are developing or running (unit) tests. The object graph has meta data that knows these settings because the context that "I am developing" or "I am running tests" is generally useful.

This line just means that the Flask application will be created with this same context.